util-core/src/main/scala/com/twitter/util/Closable.scala

```diff
@@ -45,6 +45,14 @@ abstract class AbstractClosable extends Closable
  */
 object Closable {
   private[this] val logger = Logger.getLogger("")
+  private[this] var collectorThreadEnabled = true
```

Comment:
What do you think about making this volatile? I would expect the callers of `stopCollectClosablesThread` to not be the "CollectClosables" thread itself, and so there may be memory visibility issues here.

Reply:
Could also make it an AtomicBoolean.

Reply:
AtomicBoolean already uses 'volatile' variable